packages/experimental/ActivityIndicator/src/ActivityIndicator.tsx

```diff
@@ -69,30 +57,26 @@ export const ActivityIndicator = compose<ActivityIndicatorType>({
 
     // React Native ActivityIndicator still takes up space when hidden, so to perfectly match would use opacity
     // hiding opacity makes the screen reader on iOS and Android skip over it
-    const hideOpacity = memoizedActivityIndicatorData.animating == false && memoizedActivityIndicatorData.hidesWhenStopped == true ? 0 : 1;
+    const hideOpacity = animating == false && hidesWhenStopped == true ? 0 : 1;
 
-    // Depends on size, line thickness and color after memoization so can't be in the memo itself
     const path = getActivityIndicatorPath(
-      memoizedActivityIndicatorData.size,
-      memoizedActivityIndicatorData.lineThickness,
-      memoizedActivityIndicatorData.activityIndicatorColor as string,
+      slotProps.root['size'],
+      slotProps.root['lineThickness'],
+      slotProps.root['activityIndicatorColor'],
```

Comment:
slotProps.root.size doesn't work here?

Reply:
No, since size is not a normal prop in ViewProps and root is of type ViewProps.

Reply:
Ah ok.. we're passing in values that don't exist on the type of `slotProps.root`. I think the fix there would be to change the type from `ViewProps` to `ViewProps & ActivityIndicatorTokens` 
``` 
export interface ActivityIndicatorSlotProps {
    root: ViewProps & ActivityIndicatorTokens;
    svg: Animated.AnimatedProps<SvgProps>;
 }
 ```

Reply:
Yeah, that would work. I would then change the sizes from small, medium, etc. to the actual numbers inside of ActivityIndicator.tsx instead of within styling.